geth/jail/jail_test.go

```diff
@@ -115,6 +116,44 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+	var wg sync.WaitGroup
+	txSendAsync := `
+		_status_catalog.commands.send({
+			"from": "` + TestConfig.Account1.Address + `",
+			"to": "` + TestConfig.Account2.Address + `",
+			"value": "0.000001"
+		})
+	`
```

Comment:
Can we just pass it as an argument instead of declaring a new variable? This piece of code isn't big and won't affect readability. Conversely, you won't have to move your eyes from where you're reading to the variable declaration.